rails-commenting.rb

```diff
@@ -6,25 +6,26 @@
 
 # FILE: app/controller/blog_posts_controller.rb
 
-# ---1)
+# ---1) This class, show a class which has been inherited from the ApplicationController
 class BlogPostsController < ApplicationController
+
   def index
-    # ---2)
+    # ---2) This method is a Get request that will return all the instances of BlogPost. 
```

Comment:
While this method will be called as a result of a get request the action here is actually an active record query.